src/aics-image-viewer/components/CellViewerCanvasWrapper/index.jsx

```diff
@@ -55,7 +55,7 @@ export default class ViewerWrapper extends React.Component {
     return (
       <div className="cell-canvas" style={{ ...STYLES.viewer, height: appHeight }}>
         <div ref={this.view3dviewerRef} style={STYLES.view3d}></div>
-        <BottomPanel>
+        <BottomPanel title="Clipping">
```

Comment:
I doubt we will ever use "Options" for this section.  On the other hand, I'm not sure "Clipping" makes so much sense when you are in 2d mode. but so be it!